Inner join not implemented

Repro:
```
>>> import cudf, dask_cudf
>>> x = cudf.DataFrame({'a': [1, 2]})
>>> x
<cudf.DataFrame ncols=1 nrows=2 >
>>> y = dask_cudf.from_cudf(x, npartitions=1)
>>> y
<dask_cudf.DataFrame | 1 tasks | 1 npartitions>
>>> y.merge(y, on=('a',), how='inner')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/rapids/lib/python3.7/site-packages/dask_cudf-0+unknown-py3.7.egg/dask_cudf/core.py", line 255, in merge
    assert how == "left", "left join is impelemented"
AssertionError: left join is impelemented
```

--- a/dask_cudf/core.py
+++ b/dask_cudf/core.py
@@ -247,7 +247,6 @@ class DataFrame(_Frame, dd.core.DataFrame):
     def merge(self, other, on=None, how="left", lsuffix="_x", rsuffix="_y"):
         """Merging two dataframes on the column(s) indicated in *on*.
         """
-        assert how == "left", "left join is impelemented"
         if on is None:
             return self.join(other, how=how, lsuffix=lsuffix, rsuffix=rsuffix)
         else:
@@ -314,10 +313,10 @@ class DataFrame(_Frame, dd.core.DataFrame):
 
             for k, dtype in rhs_dtypes:
                 data = np.zeros(len(lhs), dtype=dtype)
-                mask_size = cudf.utils.calc_chunk_size(
-                    data.size, cudf.utils.mask_bitsize
+                mask_size = cudf.utils.utils.calc_chunk_size(
+                    data.size, cudf.utils.utils.mask_bitsize
                 )
-                mask = np.zeros(mask_size, dtype=cudf.utils.mask_dtype)
+                mask = np.zeros(mask_size, dtype=cudf.utils.utils.mask_dtype)
                 sr = cudf.Series.from_masked_array(
                     data=data, mask=mask, null_count=data.size
                 )

--- a/dask_cudf/join_impl.py
+++ b/dask_cudf/join_impl.py
@@ -50,7 +50,6 @@ def join_frames(left, right, on, how, lsuffix, rsuffix):
     lsuffix, rsuffix : str
 
     """
-    assert how == "left"
 
     def fix_left(df):
         newdf = cudf.DataFrame()
@@ -77,20 +76,7 @@ def join_frames(left, right, on, how, lsuffix, rsuffix):
     )
 
     def merge(left, right):
-        if left is None and right is None:
-            # FIXME: this should go inside cudf so it can merge two empty
-            #        frames
-            return empty_frame
-        elif left is None:
-            # FIXME: this should go inside cudf so it can merge empty frames
-            #        left frames
-            return empty_frame
-        elif right is None:
-            # FIXME: this should go inside cudf so it can merge empty frames
-            #        right frames
-            return fix_left(left)
-        else:
-            return left.merge(right, on=on, how=how, lsuffix=lsuffix, rsuffix=rsuffix)
+        return left.merge(right, on=on, how=how, lsuffix=lsuffix, rsuffix=rsuffix)
 
     left_val_names = [k for k in left.columns if k not in on]
     right_val_names = [k for k in right.columns if k not in on]

--- a/dask_cudf/tests/test_join.py
+++ b/dask_cudf/tests/test_join.py
@@ -3,6 +3,7 @@ from functools import partial
 import numpy as np
 import pytest
 
+import cudf
 import cudf as gd
 import dask_cudf as dgd
 import dask.dataframe as dd
@@ -204,3 +205,21 @@ def test_merge_1col_left(left_nrows, right_nrows, left_nkeys, right_nkeys, how="
     got = got.sort_values(["x", "a_x", "a_y"]).reset_index(drop=True)
 
     dd.assert_eq(expect, got)
+
+
+@pytest.mark.parametrize("how", ["left", "inner"])
+def test_how(how):
+    left = cudf.DataFrame({"x": [1, 2, 3, 4, None], "y": [1.0, 2.0, 3.0, 4.0, 0.0]})
+    right = cudf.DataFrame({"x": [2, 3, None, 2], "y": [20, 30, 0, 20]})
+
+    dleft = dd.from_pandas(left, npartitions=2)
+    dright = dd.from_pandas(right, npartitions=3)
+
+    expected = left.merge(right, how=how, on="x")
+    result = dleft.merge(dright, how=how, on="x")
+
+    dd.assert_eq(
+        result.compute().to_pandas().sort_values("x"),
+        expected.to_pandas().sort_values("x"),
+        check_index=False,
+    )

